product_details/storage.py

```diff
@@ -181,3 +181,50 @@ def update(self, name, content, last_modified):
 
         with open(lm_fn, 'w') as lm_fo:
             lm_fo.write(last_modified)
+
+    def all_json_files(self):
+        json_files = []
+        for root, dirs, files in os.walk(self.json_dir):
+            root = os.path.relpath(root, self.json_dir)
+            root = '' if root == '.' else root
+            json_files.extend(os.path.join(root, f) for f in files if f.endswith('.json'))
+
+        return json_files
+
+
+def json_file_data_to_db(model=None):
+    """Import JSON file data into the DB.
+
+    This is a function to be used in a data migration.
+    It's here mostly so that it can be imported and tested.
+    """
+    PDModel = model or ProductDetailsFile
+    if PDModel.objects.count():
+        # nothing to do if there's already data
+        return
+
+    default_path = settings_defaults.PROD_DETAILS_DIR
+
+    storage = PDFileStorage()
+    files = storage.all_json_files()
+
+    if not files:
+        if storage.json_dir == default_path:
+            # no files to load
+            return
+
+        storage = PDFileStorage(json_dir=default_path)
+        files = storage.all_json_files()
+
+    if not files:
+        return
+
+    for filename in files:
+        PDModel.objects.create(
```

Comment:
NIT: this would likely benefit from using [bulk_create](https://docs.djangoproject.com/en/1.8/ref/models/querysets/#bulk-create).